replication_chexnet_cuda_local.py

```diff
@@ -212,16 +209,17 @@ def train_model(model, train_loader, val_loader, n_epochs, logfile):
 
             gc.collect()
             torch.cuda.empty_cache()
+
             model.eval()
             log.write('AUROCs on validation dataset:\n')
-            val_loss = 0           
+            test_loss = 0           
```

Comment:
rename

Reply:
I feel that lines 215, 217, and 222 should be named as val_loss, since this is the loss on the validation dataset. we need to distinguish the test loss and val loss in this step.

Reply:
Please don't rename it, val_loss is for calculating the validation loss from the validation test, we need this value in the scheduler step, here is the related code:
                val_loss = eval_model(model, val_loader, logfile)

            log = open(logfile, "a")
            log.write('Epoch: {} \tLearning Rate for first group: {:.10f}\n'.format(epoch+1, optimizer.param_groups[0]['lr']))
            model.train()
            scheduler.step(val_loss)

Reply:
Done